src/open-scd.ts

```diff
@@ -121,17 +121,13 @@ export class OpenSCD extends Waiting(
 
     mwc-linear-progress {
       position: fixed;
+      --mdc-linear-progress-buffer-color: var(--primary);
       --mdc-theme-primary: var(--mdc-theme-secondary);
       left: 0px;
-      top: 112px;
+      top: 0px;
       width: 100%;
       pointer-events: none;
-    }
-
-    @media (max-width: 599px) {
-      mwc-linear-progress {
-        top: 104px;
-      }
+      z-index: 10;
```

Comment:
```suggestion
      z-index: 1000;
```

just to be safe, we want this to be visible whatever other plugins do.

Reply:
Sure, thanks.